package.json

```diff
@@ -139,9 +139,15 @@
     "webpack": "^4.44.2"
   },
   "peerDependencies": {
+    "@types/react": ">=16.9.35",
```

Comment:
I think the initial step is just to change the normal dep to `>=` range to allow the 17 types and enable deduping. That is if we are actually compatible with those types

Reply:
Is there a way for us to test this locally to make sure dedupe works as expected?

Reply:
no, but it should, has up until now at least